bootcfg/server/serverpb/rpc.proto

```diff
@@ -19,6 +19,27 @@ service Profiles {
   rpc ProfileList(ProfileListRequest) returns (ProfileListResponse) {};
 }
 
+service Select {
+  rpc SelectGroup(SelectGroupRequest) returns (SelectGroupResponse) {};
```

Comment:
There should be several methods like these. SelectGroup, SelectProfile, SelectIgnition, SelectCloudConfig. Basically machine labels (usually from query parameters) are used to select the first matching Group, Profile, Ignition Config, etc.